docs/source/en/pipeline_tutorial.mdx

```diff
@@ -257,4 +259,37 @@ sudo apt install -y tesseract-ocr
 pip install pytesseract
 ```
 
-</Tip>
\ No newline at end of file
+</Tip>
+
+## Using `pipeline` on large models with 🤗 `accelerate`:
+
+You can easily run `pipeline` on large models using 🤗 `accelerate`! First make sure you have installed `accelerate` with `pip install accelerate`. 
+
+Let's assume you fullfill the hardware requirements to run a large model such as `bloom` (which has 176B parameters, so ~350GB in `bfloat16`). First load your model
+using `device_map="auto"`
+
+```py
+# pip install accelerate
+import torch
+from transformers import AutoModelForCausalLM, AutoTokenizer, pipeline
+
+model = AutoModelForCausalLM.from_pretrained("bigscience/bloom", torch_dtype=torch.bfloat16, device_map="auto")
+tokenizer = AutoTokenizer.from_pretrained("bigscience/bloom")
```

Comment:
Maybe use a smaller example and say in a note the user can replace it by BLOOM?

Reply:
Sure!